src/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketException.cs

```diff
@@ -122,11 +122,6 @@ public WebSocketException(string message, Exception innerException)
         {
         }
 
-        public override void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
Technically it's a breaking change to remove a public override like this, right? Regardless, if we are removing it, it should be removed from the ref, too.